src/csharp/BUILD-INTEGRATION.md

```diff
@@ -389,12 +398,17 @@ and "GrpcServices" metadata, but it might be useful in situations where you want
 to explicitly pass some otherwise unsupported (e.g. experimental) options to the
 `grpc_csharp_plugin`.
 
+* __AdditionalImportDirs__
+Pass additional directories to include proto files.  These may be directories _outside_ of the 
```

Comment:
IMHO it's inaccurate to say "Pass additional directories to include proto files".
How about "Specify additional directories in which the protocol buffer compiler will search for imported proto files."?

FTR, the official helptext for the `-I` flag in `protoc` is : *Specify the directory in which to search for imports.  May be specified multiple times; directories will be searched in order. ...*

Reply:
updated